entry.go

```diff
@@ -42,7 +42,7 @@ func NeverExpires() time.Time {
 
 // PreserveExpiration : return a time value corresponding to quasardb preserve expiration value
 func PreserveExpiration() time.Time {
-	return time.Unix(0, C.qdb_preserve_expiration)
+	return time.Unix(0, -1)
```

Comment:
There wasn't any other way but to put `-1` literally?

Reply:
Trying to put something like C.longlong(C.qdb_preserve_expiration) would throw with go 1.9
I can create a const value like this:
const PreserveExpiration time.Time = time.Unix(0, -1)